docker/grafana/datasource.local.yml

```diff
@@ -4,6 +4,6 @@ datasources:
   - name: Prometheus
     type: prometheus
     access: proxy
-    url: http://localhost:9090
+    url: http://host.docker.internal:9090
```

Comment:
Duplicate these files or you'll break the linux setup

Reply:
No, doesn't work for me

```
$ curl host.docker.internal:80
curl: (6) Could not resolve host: host.docker.internal
```

Reply:
> No, doesn't work for me

Dang! :/

I have updated with separate config files for mac